autonomous_software_pkg/src/controller.py

```diff
@@ -137,9 +143,13 @@ def __init__(self, gui_flag):
                 ControllerDebugInfo,
                 queue_size=10,
             )
+           
 
     # ##################### PURE PURSUIT FUNCTIONS #####################
 
+    def callback_speed_control(self, msg):
+        self.THROTTLE_BASELINE = int(msg.data)
```

Comment:
This is not super clean: if we want to store the value from the message, we should change the name of the variable to not look like a constant anymore. Also, I think we don't need to cast it to int since it's already an int in the message.

Reply:
Good point, thank you.

renamed variable to "throttle_value" and removed cast.

Reply:
Great, thx. Even better would be to still keep the class constant, and replace line 75 by `self.throttle_value = self.THROTTLE_BASELINE`, so that it does not look like the `100` is coming magically.